pyplanet/utils/style.py

```diff
@@ -1,14 +1,12 @@
 import re
-import struct
 
-import binascii
 
-STRIP_ALL = dict(letters='wnoitsgz<>', part=r'\$[lh]\[.+\]|\$[lh]|\$[0-9a-f]{3}')
+STRIP_ALL = dict(letters='wnoitsgz<>', part=r'(?<!\$)\$[lh]\[.+\]|(?<!\$)\$[lh]|(?<!\$)\$[0-9a-f]{3}|(?<!\$)\$[f-y]{1}')
```

Comment:
I think the last range should be `[a-y]` instead of `[f-y]`

Reply:
This I have resolved now in the updated commit.